### PR TITLE
SFD 191 Update TCS Navigation

### PR DESCRIPTION
**Pull request by jmain-scottlogic**
Updated TCS navigation to follow hierarchy in [ticket](https://scottlogic.atlassian.net/browse/SFD-191), screenshot bellow 
image

- Restructured page folder so in same layout as new hierarchy
- Created sub nav pop down menu for desktop and expandable menus on mobile
- Created generic side nav bar layout page
- Fixed some typos
- Made links all same format 
- Created some new landing pages
- Set up redirects from old routes

Also fixes 
- SFD-190 TCS navigation active state
- SFD-180 Links to worked examples on Hardware life Cycle page broken

### CODE REVIEW

File: pages/case_studies/case_studies.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+title: Case Studies
+permalink: /case-studies
+---
```
Comment (jmain-scottlogic) [inline pages/case_studies/case_studies.md:5]: Assuming this and GWF page should not published? Assuming content will come from gwf-case-study branch.

File: pages/resources/links.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+title: Links
+permalink: /resources/links
+published: true
```
Comment (jmain-scottlogic) [inline pages/resources/links.md:5]: Set to publish false as no content

File: pages/resources/tools.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+title: Tools
+permalink: /resources/tools
+published: true
```
Comment (jmain-scottlogic) [inline pages/resources/tools.md:5]: Set to publish false as no content

File: pages/resources/references.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+title: Links
+permalink: /resources/references
+published: false
```
Comment (jmain-scottlogic) [inline pages/resources/references.md:5]: Set to publish false as no content

File: pages/technology_categories/technology_categories.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+layout: sideNavigation
+title: Technology Categories
+permalink: /technology-categories
+redirect_from:
+    - /information
+---
+
+# Technology Categories
```
Comment (jmain-scottlogic) [inline pages/technology_categories/technology_categories.md:9]: Probably need some better content for this page.

File: pages/case_studies/case_studies.md (hunk 1 above)
Comment (dkrees) [reply pages/case_studies/case_studies.md:5]: Not yet no, we'll merge all that in when it's ready.

I also don't think we need sub-navigation for case studies. Just a link to a case studies page which can list the case studies with a bit of blurb about it, linking to each case study from there.

Comment (jmain-scottlogic) [reply pages/case_studies/case_studies.md:6]: Have set case studies pages to be unpublished and removed from navigation. Also removed the sub nav for case studies

File: pages/technology_categories/technology_categories.md (hunk 5 above)
Comment (jmain-scottlogic) [reply pages/technology_categories/technology_categories.md:9]: Given no previous page for information, seem a basic list of sub page will suffice. Can add more at later date if needed

**Comment by sdun-scottlogic** [comment]
Tested & Approved - SD.

**Pull request merged**